client/src/components/Chat.jsx

```diff
@@ -482,7 +482,7 @@ const Chat = () => {
 												}`}
 											>
 												{containsBadword && !isSender && !badwordChoices[id] ? (
-													<div className="flex flex-col border-red border w-full rounded-r-md p-3">
+													<div className="dark:text-white text-black flex flex-col border-red border w-full rounded-r-md p-3">
```

Comment:
So I know I gave you the example here, but I'm sure you actually tested how it would look like.

Compared to the dark-mode. Doing this here makes all the texts white including what we need to be red.

The user has two possible options which is hide the message or see it. They have two different text colours and this code here keeps them the same.

Which isn't what we want

Reply:
Okay gotcha let me get that fixed.

Reply:
I might be confused. When I look at them in light and dark mode, this is how it looks:

<img width="342" alt="badword_darkmode" src="https://github.com/user-attachments/assets/d5641b80-4db6-4bbf-87e3-0a0576006ee4">
<img width="334" alt="badword_lightmode" src="https://github.com/user-attachments/assets/8c3aa6f9-abfb-4486-80e7-b476768de4fd">

The two options for "See" and "Hide" are different colors on my end (Black and Red for the light mode).

Let me know if I am misunderstanding and looking in the wrong area.

Reply:
Na, it's my fault. I forgot the other color set will override the one at the top.